### DISCUSSION TITLE
How to change scheduler logging level?

### DISCUSSION THREAD

**Discussion by uplsh580** [apache/airflow#17108] [Q&A] [answered]
In my Production environment, the Scheduler's logging takes up a large amount of capacity, so I want to adjust the Scheduler's logging level (INFO -> WARN).

Is there a way to adjust only Scheduler's logging level while maintaining TASK logging level?

FYI. My Airflow Version is 2.1.0

**Comment by potiuk** [comment, marked as answer]
Yep. Look there https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/logging-architecture.html . Airflow has logging configuration that you can modify (see here for the best way of doing it https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/logging-tasks.html#advanced-configuration) - you can take a look at the DEFAULT_LOGGING_CONFIG https://github.com/apache/airflow/blob/main/airflow/config_templates/airflow_local_settings.py#L57 and modify it accordingly.

This is standard Pyhon Logging functionality so you can fully configure it as you want. https://docs.python.org/3/library/logging.html

It's best if you use the approach suggestedi itne "advanced configuration" link above - deepcopy original configuration and apply your changes - this will allow you to keep it updated when it changes in the future.
